Player feature to Dev

GuidoYebara opened the pull request:
-Fixed camera, now follows the player without stuttering
-Fixed player movement. It now walks and does that on all directions, with the correct animations.
-Add new the new Unity Controller Interface and created PlayerControls asset to read input.

GuidoYebara committed:
Update Player.cs

Player Feature created
Vector3 for movement

GuidoYebara committed:
Camera/New Controls Interface

-Camera Fixed. Sticks to the player. Can be improved with cinemachine
-Player movement with different animations
-Added New Input Manager and PlayerControls for movement

0Lezz0 commented on Assets/Scripts/Camera.cs:
```diff
@@ -31,7 +31,9 @@ void Update()
-        
+        transform.position = pos;
+
+        //Vector3.SmoothDamp(transform.position, pos, ref velocity, smooth);
```
Estaría bueno sacar el código comentado si vamos a ir a dev

GuidoYebara committed:
Player.cs + Controls

-Replaced old Input system mouse coordinates with the new. (mousePos variable)
-Added in the new Input system: main spell, secondary spell.
-Created cast spell method
